Packs/UrlScan/Integrations/UrlScan/UrlScan.py

```diff
@@ -263,13 +306,13 @@ def format_results(client, uuid):
     # Scan Lists sometimes returns empty
     num_of_attempts = 0
     relationships = []
-    response = urlscan_submit_request(client, uuid)
+    response, metric_results, _ = urlscan_submit_request(client, uuid)
```

Comment:
is this being used?

Reply:
It gets passed up to the submission function so we know whether or not to schedule the request.